src/EventListener/LogRevisionsListener.php

```diff
@@ -582,6 +582,12 @@ private function saveRevisionEntityData(EntityManagerInterface $em, ClassMetadat
             );
         }
 
+        foreach ($params as $key => $parameterValue) {
+            if (interface_exists(\BackedEnum::class) && $parameterValue instanceof \BackedEnum) {
```

Comment:
Do we really need the `interface_exists()` check?

Reply:
I don't know how PHP behaves when doing an instanceof check against an interface that does not exist, even if that maybe doesn't emit a warning today it doesn't mean it's not going to in the future so I think it's better to keep this as is as it definitively doesn't hurt.

Reply:
PHP handles the absence of the class gracefully. Avoiding superfluous function calls is always desirable.
```suggestion
            if ($parameterValue instanceof \BackedEnum) {
```

Reply:
Done